clever/src/selfcheck.py

```diff
@@ -715,6 +715,41 @@ def check_network():
             failure('Not found %s in /etc/hosts, ROS will malfunction if network interfaces are down, https://clever.coex.tech/hostname', ros_hostname)
 
 
+@check('Hardware health')
+def check_hw_health():
```

Comment:
I would name it RPi checks, because they are only RPi related.

Reply:
What about other hardware, though? Wouldn't we want to add additional checks here eventually?

Reply:
Since we have `cpu_usage` and `boot_duration`, these checks are pretty granulated. So I'm not sure we're gonna group them a lot.

Reply:
My idea also is that the user can blacklist some of the checks when use this as a service.

Reply:
Sounds fair, will change to "RPi health"